spec/services/dialog_field_visibility_service_spec.rb

```diff
@@ -0,0 +1,38 @@
+describe DialogFieldVisibilityService do
+  let(:subject) { described_class.new }
+
+  describe "#determine_visibility" do
+    let(:subject) do
+      described_class.new(
+        auto_placement_visibility_service,
+      )
+    end
+
+    let(:options) do
+      {
+        :auto_placement_enabled          => auto_placement_enabled,
+      }
+    end
+
+    let(:auto_placement_visibility_service) { double("AutoPlacementVisibilityService") }
+    let(:auto_placement_enabled) { "auto_placement_enabled" }
+
+    before do
```

Comment:
Since there is only one `it`, wouldn't it be faster to move this setup into the `it`?